tests/virtual_machines_test.go

```diff
@@ -85,6 +85,18 @@ var _ = Describe("[rfe_id:3503][crit:medium][vendor:cnv-qe@redhat.com][level:com
 		})
 
 		Context("When Running with default opt-mode configuration", func() {
+			BeforeEach(func() {
+				By("Getting the current VM Opt-mode")
+				vmWebhookOptMode, err := getOptMode(vmNamespaceOptInLabel)
+				Expect(err).ToNot(HaveOccurred(), "Should succeed getting the mutating webhook opt-mode")
+
+				if vmWebhookOptMode == optInMode {
+					By("opting in the namespace")
+					err := addLabelsToNamespace(TestNamespace, map[string]string{vmNamespaceOptInLabel: "allocate"})
```

Comment:
Maybe we could make a constant for `allocate`, as it's used in two places

Reply:
Done